src/main/java/tracko/logic/commands/order/FindOrderCommand.java

```diff
@@ -1,12 +1,15 @@
 package tracko.logic.commands.order;
 
 import static java.util.Objects.requireNonNull;
+import static tracko.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static tracko.logic.parser.CliSyntax.PREFIX_ITEM;
+import static tracko.logic.parser.CliSyntax.PREFIX_NAME;
 
 import tracko.commons.core.Messages;
 import tracko.logic.commands.Command;
 import tracko.logic.commands.CommandResult;
 import tracko.model.Model;
-import tracko.model.order.OrderContainsKeywordsPredicate;
+import tracko.model.order.OrderMatchesFlagsAndPrefixPredicate;
 
 /**
  * Finds and lists all orders whose name contains any of the argument keywords.
```

Comment:
Since the functionality of the FindOrderCommand has been updated, I think it might also be good to update the JavaDocs to give better context